src/languages/en.js

```diff
@@ -274,8 +275,8 @@ export default {
         markAsUnread: 'Mark as unread',
         markAsRead: 'Mark as read',
         editComment: 'Edit comment',
-        deleteAction: ({action}) => `Delete ${ReportActionsUtils.isMoneyRequestAction(action) ? 'request' : 'comment'}`,
-        deleteConfirmation: ({action}) => `Are you sure you want to delete this ${ReportActionsUtils.isMoneyRequestAction(action) ? 'request' : 'comment'}?`,
+        deleteAction: ({isMoneyRequest}) => `Delete ${isMoneyRequest ? 'request' : 'comment'}`,
+        deleteConfirmation: ({isMoneyRequest}) => `Are you sure you want to delete this ${isMoneyRequest ? 'request' : 'comment'}?`,
```

Comment:
Good catch. The reason I made this change was to remove a dependency cycle, because requiring ReportActionsUtils in en.js/es.js causes a huge cycle. I'll see if I can change this back

Reply:
Refactored so that ReportActionsUtils does not require ReceiptUtils!